src/shared/ui/SelectTabs/SelectTabs.jsx

```diff
@@ -0,0 +1,44 @@
+import "./SelectTabs.css";
+import cx from "classnames";
+
+
+const genresList = [
```

Comment:
we still have mock data in a ui component, it should be in a separate place, please, look at the structure again 
<img width="221" alt="image" src="https://github.com/AnnaMartis/Epam-project/assets/36797396/e8334c3b-f0e1-44c7-9192-81d03c85eff2">

Reply:
I guess I've got the point and I moved it to another file named constants.

Reply:
please, take a look at the picture :) I literally showed where we could store it (near SelectTabs.js)

Reply:
okay sorry )) 
I have just a question, we start it with small letter as it is not a component right?

Reply:
done

Reply:
> okay sorry )) I have just a question, we start it with small letter as it is not a component right?

right